### DISCUSSION TITLE
No counters when running

### DISCUSSION THREAD

**Discussion by Krwak** [cataclysmbn/Cataclysm-BN#3804] [General]
### Is your feature request related to a problem? Please describe.

When running away from enemies and trying to conserve stamina, you don't want to counter enemy attacks.
Right now the only way to avoid this is to switch to a fighting style that doesn't have any counter.
Another reason is that countering while running doesn't make sense.

### Describe the solution you'd like

Remove the ability to counter when running.
This will have two benefits.
1. Conserve stamina for avoiding enemies.
2.  Gives players more control over the counters in general.

### Additional context

![image](https://github.com/cataclysmbnteam/Cataclysm-BN/assets/72284934/ce0d03bd-01dd-43e5-a88f-7ad251eb6f05)

**Comment by Zireael07** [comment]
Note: image didn't upload, probably because you clicked away from the box while it was uploading
